core/orch_test.go

```diff
@@ -1349,7 +1675,10 @@ func TestPriceInfo_TxMultiplierError_ReturnsError(t *testing.T) {
 func TestDebitFees(t *testing.T) {
 	n, _ := NewLivepeerNode(nil, "", nil)
 	n.Balances = NewAddressBalances(5 * time.Second)
-	orch := NewOrchestrator(n)
+	rm := &stubRoundsManager{
```

Comment:
Can most of these `stubRoundsManager` constructor invocations be empty except for the case where `ProcessPayment()` is invoked in the test?

Reply:
fixed in 35d8e94a